src/content/docs/ar/guides/fonts.mdx

```diff
@@ -0,0 +1,146 @@
+---
+title: استخدام الخطوط المخصصة
```

Comment:
Oh and one small additional thing. Reviewing this PR prompted me to make this small change to the English sidebar label: #10717.

We should probably do the same here, something like:
```suggestion
title: استخدام الخطوط المخصصة
sidebar:
  label: الخطوط
```
(Feel free to correct my Arabic, of course 😁)